modules/serverless-config-replicas-serving.adoc

```diff
@@ -41,3 +41,37 @@ spec:
   high-availability:
     replicas: 3
 ----
+
+. You can also specify the number of replicas for a specific workload, which overrides the global setting for Knative Serving:
++
+.Example YAML
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  high-availability:
+    replicas: 3
+  workloads:
+  - name: webhook
+    replicas: 4
```

Comment:
workloads setting takes precedence, maybe something worth noting.

Reply:
@skonto :+1: Made it more explicit using a NOTE.